integration/tests/test_utils.py

```diff
@@ -229,18 +233,16 @@ def success_predicate(result):
     return spin(fn, success_predicate).json()
 
 
-def _nested_dict_merge(a, b, path=None):
```

Comment:
Was there a bug in this function? If so, we may have to fix it in a few other codebases: dcos-cassandra, etc.